apps/extension/docs/extension-CI-setup.md

```
@@ -0,0 +1,209 @@
+  - 📤 Publishes to VS Code Marketplace
+  - 🌍 Publishes to Open VSX Registry
+  - 📊 Updates package versions and CHANGELOG
+
+## 🚀 Changeset Workflow
+
+### Creating Changes
+When making changes to the extension:
+
+1. **Make your code changes**
+2. **Create a changeset**:
+   ```bash
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)